Do not automatically build against Cocoa on macOS.

At Prefix/macOS, we do not support the propietary Cocoa SDK.  GUI
applications use X11.

The patch is specific to our need and not forwarded upstream.

Signed-off-by: Benda Xu <heroxbd@gentoo.org>

Index: matplotlib-3.7.2/setup.py
===================================================================
--- matplotlib-3.7.2.orig/setup.py
+++ matplotlib-3.7.2/setup.py
@@ -49,7 +49,6 @@ mpl_packages = [
     setupext.FreeType(),
     setupext.Qhull(),
     setupext.Tests(),
-    setupext.BackendMacOSX(),
     ]
 
 
